workflow-service/src/main/java/com/redhat/parodos/workflow/execution/continuation/AsyncWorkFlowContinuer.java

```diff
@@ -0,0 +1,11 @@
+package com.redhat.parodos.workflow.execution.continuation;
+
+import java.util.UUID;
+
+import com.redhat.parodos.workflows.work.WorkContext;
+
+public interface AsyncWorkFlowContinuer {
+
+	void execute(String projectId, String workflowName, WorkContext workContext, UUID executionId);
```

Comment:
The @Async annotation hides the fact that this method is called asynchronously. 
So my next thought is why not return a `Future<WorkReport>` in the signature to express that?

Reply:
There are two implementations of the Async method. The other one expects to return a ComputableFuture if you need to return a handle for the result.
In this context, we don't need to return any response or a feedback. Just to spawn the execution and move to the next one.

Reply:
what about move/copy the @Async from impl to here and rename the method to executeAsync(...)?